### PR TITLE
fix: renamed all scriptConf arguments to script_conf

### PR DESCRIPTION
**Pull request by Fagtoy**
renamed all scriptConf arguments to script_conf in order to generalize script arguments to avoid errors with calling scripts with an invalid argument

### CODE REVIEW

File: parseAndPopulate/resolve_expiration.py [hunk 1]
```diff
@@ -55,10 +51,8 @@ def __init__(self):
         super().__init__(help, None, None if __name__ == '__main__' else [])
 
 
-def main(scriptConf=None):
+def main(script_conf: t.Optional[BaseScriptConfig] = None):
```
Comment (richardzilincikPantheon) [inline parseAndPopulate/resolve_expiration.py:54]: If we never initialize a `ScriptConfig` object the `--help` argument will be ignored.

File: parseAndPopulate/reviseSemver.py [hunk 2]
```diff
@@ -86,10 +86,8 @@ def load_from_json(path: str):
         return json.load(reader)
 
 
-def main(scriptConf=None):
+def main(script_conf: t.Optional[BaseScriptConfig] = None):
```
Comment (richardzilincikPantheon) [inline parseAndPopulate/reviseSemver.py:89]: same as above

File: utility/reviseTreeType.py [hunk 3]
```diff
@@ -47,10 +48,8 @@ def __init__(self):
         super().__init__(help, None, [])
 
 
-def main(scriptConf=None):
+def main(script_conf: t.Optional[BaseScriptConfig] = None):
```
Comment (richardzilincikPantheon) [inline utility/reviseTreeType.py:51]: same here

**Review by richardzilincikPantheon — CHANGES_REQUESTED** [review]

File: parseAndPopulate/resolve_expiration.py (hunk 1 above)
Comment (Fagtoy) [reply parseAndPopulate/resolve_expiration.py]: thanks, fixed

**Review by richardzilincikPantheon — APPROVED** [review]

**Review by SlavomirMazurPantheon — APPROVED** [review]

**Pull request merged**